geode-redis/src/main/java/org/apache/geode/redis/GeodeRedisServer.java

```diff
@@ -596,61 +596,121 @@ public Thread newThread(Runnable r) {
 
     bossGroup = null;
     workerGroup = null;
+
     Class<? extends ServerChannel> socketClass;
     if (singleThreadPerConnection) {
       socketClass =
-          startRedisServiceSingleThreadPerConnection(selectorThreadFactory, workerThreadFactory);
+          startRedisServiceSingleThreadPerConnection(
+              selectorThreadFactory,
+              workerThreadFactory);
     } else {
-      bossGroup = new NioEventLoopGroup(numSelectorThreads, selectorThreadFactory);
-      workerGroup = new NioEventLoopGroup(numWorkerThreads, workerThreadFactory);
+      bossGroup =
+          new NioEventLoopGroup(numSelectorThreads, selectorThreadFactory);
+      workerGroup =
+          new NioEventLoopGroup(numWorkerThreads, workerThreadFactory);
+
```

Comment:
I realize this is perhaps a very personal preference, but I really dislike breaking up assignments (or method calls) like this when the line itself isn't really very long and doesn't already hit the max line length for code style. I find it makes methods longer such that I end up having to scroll back and forth to read the method.